test/atomic_enum_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class TestAtomicEnum < MiniTest::Unit::TestCase
```

Comment:
When writing unit tests, I like to browse the class I'm testing and look at each line and think: If I delete this line of code, will a test fail? If I change this greater than to a less than, would a test fail? What if it's a less than or equal?

I bet you can find a lot of lines for which that's not true and increase test coverage.